viz/LaserLine.cpp

```diff
@@ -2,44 +2,150 @@
 #include "LaserLine.hpp"
 
 using namespace vizkit3d;
+using namespace vizkit3d_laser_line;
 
-struct LaserLine::Data {
-    // Copy of the value given to updateDataIntern.
-    //
-    // Making a copy is required because of how OSG works
-    base::samples::LaserScan data;
-};
 
+#define milimeters_to_meters(x) (x) / 1000.0
```

Comment:
just don't use #define in C++.

~~~
static inline double millimeters_to_meters(double x) { return x / 1000.0; }
~~~